doc/7/core-classes/observer-search-result/properties/index.md

```diff
@@ -0,0 +1,16 @@
+---
+code: false
+type: page
+title: Properties
+description: ObserverSearchResult class properties
+order: 100
+---
+
+# Properties
+
+| Property     | Type                  | Description                                            |
+|--------------|-----------------------|--------------------------------------------------------|
+| `hits`       | <pre>Observer[]</pre> | Page results                                           |
+| `total`      | <pre>number</pre>     | Total number of items that _can_ be retrieved          |
+| `fetched`    | <pre>number</pre>     | Number of retrieved items so far                       |
+| `notifyOnly` | <pre>boolean</pre>    | If `true`, the observers will not update there content |
```

Comment:
I don't get how this can be useful. What's the use case?

Reply:
It's one of the use case I demonstrate in the documentation. You may want to search for documents and they display them in a list but you also want them to update in realtime. 

Vue.js template: https://github.com/kuzzleio/sdk-javascript/pull/495/files#diff-ced1acc7756d77fd72a2caae0fa7b042R26
API call: https://github.com/kuzzleio/sdk-javascript/pull/495/files#diff-ced1acc7756d77fd72a2caae0fa7b042R50

Reply:
That's in the case someone only want an easy way to listen to changes happening to a document or a collection of document but they want to change the content themselves rather than having the `_source` automatically mutated
(@xbill82 )

Reply:
Yep, but I still don't get the utility of this feature, since we already have the `subscribe` method available

Reply:
So if I activate this option, I still have a `_source` content, but it is outdated as soon as I receive an update notification. And this is on purpose.

I don't see how this can be something we want to have.

I'd rather have no `_source` content at all, only send notification events, and let users update whatever object they maintain themselves.

Reply:
We need to keep the `_source` property to have directly the entire document content and not only when the first notification is received.

Also I feel it's more convenient to use directly the observer rather than writing more code:
```js
const observer = await kuzzle.observer.get(
  'nyc-open-data', 
  'yellow-taxi',
  'document-1');

this.document = observer;

// vs

const observer = await kuzzle.observer.get(
  'nyc-open-data', 
  'yellow-taxi',
  'document-1');

observer.on('change', changes => {
  this.document = { ...this.document, ...changes }
})
```